src/DurableTask.Netherite/OrchestrationService/NetheriteOrchestrationServiceSettings.cs

```diff
@@ -59,17 +60,38 @@ public class NetheriteOrchestrationServiceSettings
         /// <summary>
         /// Gets or sets the maximum number of work items that can be processed concurrently on a single node.
         /// The default value is 100.
-        /// Matches Microsoft.Azure.WebJobs.Extensions.DurableTask.
+        /// Matches corresponding property in Microsoft.Azure.WebJobs.Extensions.DurableTask.DurableTaskOptions.
         /// </summary>
         public int MaxConcurrentActivityFunctions { get; set; } = 100;
 
         /// <summary>
         /// Gets or sets the maximum number of orchestrations that can be processed concurrently on a single node.
         /// The default value is 100.
-        /// Matches Microsoft.Azure.WebJobs.Extensions.DurableTask.
+        /// Matches corresponding property in Microsoft.Azure.WebJobs.Extensions.DurableTask.DurableTaskOptions.
         /// </summary>
         public int MaxConcurrentOrchestratorFunctions { get; set; } = 100;
 
+        /// <summary>
+        /// Gets or sets the maximum number of work items that can be processed concurrently on a single node.
```

Comment:
nit: let's make this more precise by saying "entity" explicitly
```suggestion
        /// Gets or sets the maximum number of Entity work items that can be processed concurrently on a single node.
```

Reply:
I have updated the comments for activities, entities, and orchestrations.